resources.go

```diff
@@ -381,9 +381,16 @@ func Provider() tfbridge.ProviderInfo {
 					Source: "autoscaling_lifecycle_hooks.html.markdown",
 				},
 			},
-			"aws_autoscaling_notification": {Tok: awsResource(autoscalingMod, "Notification")},
-			"aws_autoscaling_policy":       {Tok: awsResource(autoscalingMod, "Policy")},
-			"aws_autoscaling_schedule":     {Tok: awsResource(autoscalingMod, "Schedule")},
+			"aws_autoscaling_notification": {
+				Tok: awsResource(autoscalingMod, "Notification"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"notifications": {
+						Type: awsType(tokens.ArrayTypePrefix+autoscalingMod+"/notificationType", "NotificationType"),
```

Comment:
Is the `tokens.ArrayTypePrefix` here necessary? And if it is, would it be better abstracted behind some kind of `awsArrayType` function that adds it automatically?

I've made the other modification to add the missing `/` to the tag.